lib/manager/components/version/FeedVersionViewer.js

```diff
@@ -218,37 +221,67 @@ class VersionSectionSelector extends Component<SelectorProps> {
       : <span><BsLabel bsStyle={color}>{text}</BsLabel></span>
   }
 
+  // Render the number of transformations applied to the current feed version
+  _renderTransformResultsCount = (version: FeedVersion) => {
+    let totalCount
+    if (version.feedTransformResult) {
```

Comment:
This is a bit of a nitpick, but couldn't we just set totalCount to 0 here and then not have to have the else statement?